activestorage/lib/active_storage/service/azure_storage_service.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-gem "azure-storage-blob", "~> 1.1"
+gem "azure-storage-blob"
```

Comment:
```suggestion
gem "azure-storage-blob", ">= 1.1"
```

Reply:
Thanks for the review. Updated to use 1.1 or higher and squashed commits.